src/cache/precomputed_quantities.jl

```diff
@@ -265,14 +265,22 @@ function set_precomputed_quantities!(Y, p, t)
     (; ᶜspecific, ᶜu, ᶠu³, ᶜK, ᶜts, ᶜp, ᶜΦ) = p
     ᶠuₕ³ = p.ᶠtemp_CT3
 
-    @. ᶜspecific = specific_gs(Y.c)
-    set_ᶠuₕ³!(ᶠuₕ³, Y)
+    @nvtx "specific_gs!" color = colorant"khaki" begin
```

Comment:
why does the annotation contain the exclamation mark (suggesting impure function?) when the actual call doesn't?

Reply:
yes, I understand that. But the actual call to `specific_gs` on line 269 doesn't have the `!` right?